python/ray/runtime_context.py

```diff
@@ -222,6 +222,7 @@ def get_actor_id(self) -> Optional[str]:
         return actor_id.hex() if not actor_id.is_nil() else None
 
     @property
+    @Deprecated(message="Use get_namespace() instead", warning=True)
```

Comment:
The reason why we create `get_xx_id` is that it returns different thing than `xx_id`.  We haven't decided whether we want to apply this to other properties yet. How about we leave this deprecation out of this PR and we can decide later whether we want to deprecate all properties.

Reply:
Sounds good.

Reply:
just wondering why we didn't change the implementation that allows `xx_id` returns a correct value.

Reply:
For backward compatibility reason.